config/config.go

```diff
@@ -76,6 +76,7 @@ type SigningProfile struct {
 	NotAfter            time.Time  `json:"not_after"`
 	NameWhitelistString string     `json:"name_whitelist"`
 	AuthRemote          AuthRemote `json:"auth_remote"`
+	CTLog               string     `json:"ct_log"`
```

Comment:
I've fixed this on my own branch - is there a way to update this PR with that commit?
Maybe best to just keep development on a branch and hold off on merging until the feature is more complete.